Added Model Settings (#220)
satisfactorymodding/Documentation: modules/ROOT/pages/Development/ExtractGameFiles.adoc

image:ExtractingGameFiles/fModelExport.png[FModel Export Configuration]

=== Specify Model Export Settings

> After specifying fModel's Custom Version and Mapping for Satisfactory, you will/may also want change model export settings.
> 
> To do so, you will need to select `Models` on the left side of the fModel settings menu.

Reviewer: ```suggestion
After specifying FModel's Custom Version and Mapping for Satisfactory you will want change model export settings so it's easier to work with exported results.
To do so, return to the Settings menu and select `Models` on the left side.
```

> From here there are several things you need to do in order to properly export models for use with modeling software such as Blender(shown in image below);

Reviewer: ```suggestion
From here there are several fields to modify in order to properly export models for use with modeling software such as Blender:
```

> * `Model Export Directory` - This can be the same or different from the output directory you chose in the previous steps.
> * `Mesh Format` - Similarly to uModel, you will want to select `glTF 2.0` as your format
> * `Level of Detail Format` - Allows you to select how many LOD's of the model to export. 
> * `Texture Format` - PNG; This allows you to export textures as PNG.

Reviewer: ```suggestion
* `Model Export Directory` - This can be the same or different from the output directory you chose in the previous steps.
* `Mesh Format` - Similarly to UModel, you will want to select `glTF 2.0` as your format
* `Level of Detail Format` - Allows you to select how many LODs (Level of Distance meshes) of the model to export. 
* `Texture Format` - Set to PNG so textures will be exported to a lossless format that supports transparency.
```

> image:ExractingGameFiles/FModelModelSettings.png[FModel Model Export Settings]

Reviewer: ```suggestion
image:ExtractingGameFiles/FModelModelSettings.png[FModel Model Export Settings]
```
